Makefile

```diff
@@ -40,51 +40,131 @@ buildnocache:
 	./ci_cd/version.sh
 	@docker-compose build --no-cache --pull
 
+build:
+	./ci_cd/version.sh
+	@docker-compose build
+
 up:
 	docker-compose up -d
 
+logs:
+	@docker-compose logs -f $(API_SERVICE)
+
 start:
 	@make up
 
+restart_api:
+	@docker-compose restart api_service
```

Comment:
From what we talked about today, for local development, I'm not clear why we'd want to restart `api_service` instead of `api_runserver`. In any case I'd do a search/replace so this is always $(API_SERVICE) when used.

Reply:
done